webapp/certified/views.py

```diff
@@ -362,6 +362,7 @@ def get_filters(request_args=None, json: bool = False):
     release_filters.extend(all_releases)
 
     if json:
+        print("TESTING JSON")
```

Comment:
Can we remove this line? is it necessary?

Reply:
Thanks for catching this! I have now updated the code to remove this line.